### PR TITLE
chore(starters): gatsby-starter-zurgbot

### PR DESCRIPTION
**Pull request by zurgbot**
## Description
Adds `gatsby-starter-zurgbot` to starter library!

https://gatsby-starter-zurgbot.netlify.com/
https://github.com/zurgbot/gatsby-starter-zurgbot

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -4130,3 +4130,27 @@
     - Mobile friendly responsive design made to be customized or leave as is
     - Separate components for everything
     - ...and more
+- url: https://gatsby-starter-zurgbot.netlify.com/
+  repo: https://github.com/zurgbot/gatsby-starter-zurgbot
+  description: The ultimate force of starter awesomeness in the galaxy of Gatsby
+  tags:
+    - Landing Page
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
```

File: docs/starters.yml [hunk 2]
```diff
@@ -4130,3 +4130,27 @@
...
+    - Linting
+    - PWA
+    - SEO
+    - Sitemap
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
```

File: docs/starters.yml [hunk 3]
```diff
@@ -4130,3 +4130,27 @@
...
     - Separate components for everything
     - ...and more
+- url: https://gatsby-starter-zurgbot.netlify.com/
+  repo: https://github.com/zurgbot/gatsby-starter-zurgbot
+  description: The ultimate force of starter awesomeness in the galaxy of Gatsby
+  tags:
+    - Landing Page
+    - Linting
+    - PWA
+    - SEO
```
Comment (LekoArts) [inline docs/starters.yml]:
```suggestion
```

Comment (colbyfayock) [reply docs/starters.yml]: curious about why this one is invalid given the solution in there to add and manage appropriate meta + the sitemap plugin working there (not to mention the sitemap tag removal change)

**Comment by colbyfayock** [comment]
not a bot, didn't realize i opened this under that account. its just the name cause `zurg` was taken and i wanted to use that 😅

File: docs/starters.yml (hunk 3 above)
Comment (LekoArts) [reply docs/starters.yml]: didn't saw the latter, it's fine then.

Comment (colbyfayock) [reply docs/starters.yml]: ah okay cool

**Comment by LekoArts** [comment]
I wasn't able to push to your branch so that's why I thought so.

However, since it would be unfair for other users who only have one account (and hence only receive swag once) I'd ask you to please open up the PR from your Colby Fayock account 'cause if we'd merge this PR you would get swag for free a second time.

Thus I'm closing the PR.

**Comment by colbyfayock** [comment]
ah thats totally fair, my bad, not the intention. here you go https://github.com/gatsbyjs/gatsby/pull/18907

**Pull request closed without merging**